pages/glossary/changes-in-content.md

```diff
@@ -4,15 +4,25 @@ key: changes-in-content
 unambiguous: true
 objective: true
 input_aspects:
+  - Accessibility tree
+  - Audio output
   - CSS styling
   - DOM tree
 ---
 
-Changes in the content of a [web page][] occur when at least one of the following occurs:
+A _change in the content_ of a [web page][] caused by an event occurs when, by comparing the [web page][] before and after the event firing, at least one of the following occurs:
```

Comment:
And timing strikes here: "after the event firing" => how long after? What happens if there is a delay between firing the event and causing the change? (that was a bit present before and becomes somewhat more obvious with this formulation…)
This is not too bad as not detecting a "change caused by an event" result in passing the rule (due to **no changes**) while failing the SC, but that does not break the fail-on-fail relationship.

Reply:
I also already had an assumption for this situation: "For the purpose of this definition, it is assumed that the changes happen instantly after the event firing and therefore the comparison between the page before and after the event firing can be made at any time after the firing of the event."

Reply:
Missed that assumption indeed 😥